components/EventsList.tsx

```diff
@@ -41,6 +41,9 @@ const EventsList = () => {
           index={0}
           enablePanDownToClose={false}
           bottomInset={BOTTOM_OFFSET}
+          onChange={(e) => {
+            console.log(e)
```

Comment:
What's this for

Reply:
I was testing to see how to get the height of the bottom sheet. I must've forgot to remove it.